src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/GroovyLanguageCoverageTest.java

```diff
@@ -0,0 +1,152 @@
+package org.jenkinsci.plugins.scriptsecurity.sandbox.groovy;
+
+import org.jenkinsci.plugins.scriptsecurity.sandbox.Whitelist;
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.GenericWhitelist;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ErrorCollector;
+import org.jvnet.hudson.test.Issue;
+
+public class GroovyLanguageCoverageTest {
```

Comment:
Was debating that but decided at least for now to keep these tests all in one place as I add them.